pkg/util/arm/resources.go

```diff
@@ -43,24 +42,29 @@ func (r ArmResource) String() string {
 // ParseArmResourceId take the resourceID of a child resource to an OpenShiftCluster
 // TODO refactor this function to support an additional layer of child resources if we ever get to that point, right now only supports 1 child resource
```

Comment:
This `TODO` comment can be removed since the function now supports an additional level of subresource nesting.